precode.sh

```diff
@@ -7,37 +7,77 @@
 #   dir3
 #       dir4
 
+mkdir /tmp/task
+mkdir /tmp/task/dir1 /tmp/task/dir2 /tmp/task/dir3
+mkdir /tmp/task/dir3/dir4
+
 # изменяем текущую директорию на task
 
+cd /tmp/task
+
 # создаём пустой файл task/dir2/empty
 
+touch /tmp/task/dir2/empty
```

Comment:
А вот тут мы уже в `./task`, так что все пути нужно писать относительно этой папки
```suggestion
touch ./dir2/empty
```

Reply:
Нужно все команды поправить